extractors/bookseries/karelians/extractors/kalastus_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class KalastusExtractor(BaseExtractor):

```

Comment:
When this and hunting extractors are added to the pipeline you might have to consider if the data is added to the correct person: primary person or spouse. Most of the time in the book entries the primary person is male and spouse is female but this is not always the case. 

You could do some assumptions on if the hunting and fishing activities were gendered back in the day. For example assume that most of the time fishing is done by males. You can see one example implementation of this kind of extractor here: https://github.com/Learning-from-our-past/kaira-core/blob/main/extractors/bookseries/karelians/extractors/martta_activity_flag_extractor.py 

In the above extractor we make assumption that only females were part of martta activities and the extractor activates only for the female persons. I'm not sure if you have to do 
```
if self._in_spouse_extractor and self._deps['person']['name']['gender'] == 'Male':
            is_female = True
        elif not self._in_spouse_extractor and self._deps['person']['name']['gender'] == 'Female':
            is_female = True
```
logic here though. Maybe you can find the current person's sex directly depending on current person's name property instead? Lumi can probably help with figuring that out.

Before making such assumptions, I'd suggest doing a quick search on text files searching the keywords (in this case "kalastus") and see if your assumption is reasonable. Then document the assumption to the extractor's doc-string and finally mentioning about it to John or other researchers. Or if you are unsure if the assumption makes sense, probably someone from researchers is happy to chip in with their opinion.

Also note that the male/female division is just an example of how to figure out which person the field here refers to. There might be other and better methods for that.